test/numbers.jl

```diff
@@ -2771,7 +2771,8 @@ let types = (Base.BitInteger_types..., BigInt, Bool,
     for S in types
         for op in (+, -)
             if S === Float16 # type instability here?
-                @test_broken @inferred Base.promote_op(op, S)
+                # broken, fixme then remove this branch
```

Comment:
No, commenting the test out doesn't tell you when you fixed the behavior - that's much more of the point of what `@test_broken` is useful for. Yes, rewriting `@test_broken @inferred` to do the right thing would be a good idea, but is also more messing with macros than I have the patience for right now.